rasa/nlu/model.py

```diff
@@ -406,5 +406,5 @@ def parse_message(self, message: Message, attribute: Text = TEXT) -> Message:
 
```

Comment:
the method name `parse_message` doesn't really make sense. Should we rename to something like `featurize_message`?

Reply:
It makes sense, but I would change it later once we settle on how it is actually used within Core.